cdc/sink/mq/manager/manager.go

```diff
@@ -19,6 +19,6 @@ package manager
 type TopicManager interface {
 	// GetPartitionNum returns the number of partitions of the topic.
 	GetPartitionNum(topic string) (int32, error)
-	// CreateTopic creates the topic.
-	CreateTopic(topicName string) (int32, error)
+	// CreateTopicAndWaitUntilVisible creates the topic and wait for the topic completion.
+	CreateTopicAndWaitUntilVisible(topicName string) (int32, error)
```

Comment:
I prefer the previous method name. wait until visible is more like an implementation detail. For me, descibe in the doc string is enough.

Reply:
This is acceptable to me because it is two different interfaces, one corresponding to the createtopic semantics of kafka and one where we require that the creation must succeed.